v2/protocol/http/protocol_test.go

```diff
@@ -222,6 +222,7 @@ func TestServeHTTP_Receive(t *testing.T) {
 	}{
 		"nil": {
 			rw:      httptest.NewRecorder(),
+			req:     httptest.NewRequest("POST", "http://unittest", nil),
```

Comment:
this has changed how https://github.com/cloudevents/sdk-go/pull/491/files#diff-255451943fd83a8be86080df4ef96fb9R263 is tested. Before the test would return m == nil because the test was testing an empty request. I made it non-nil and there is an issue.